CMakeLists.txt

```diff
@@ -94,36 +94,6 @@ if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Haiku")
     set(CMAKE_C_FLAGS_INIT "-fPIC")
 endif()
 
-#
```

Comment:
This comment should probably be left in, perhaps with "currently no C++ code" replaced with "no C++ code", to note that removing the "C" can produce weird symptoms that can take some work to diagnose.  (It also avoids doing extra C++-related work when configuring.)

Reply:
I've modified the comment in tcpdump's CMakeLists.txt, and the same change should be made to libpcap's CMakeLists.txt, so I'll just merge this change and then add the new version of the comment.